cmd/conduit/main.go

```diff
@@ -71,7 +71,16 @@ func parseConfig() conduit.Config {
 
 		connectorsDir = flags.String("connectors.path", "./connectors", "path to standalone connectors directory")
 
-		pipelinesDir = flags.String("pipelines.path", "./pipelines", "path to the directory that has the yaml pipeline configuration files, or a single pipeline configuration file")
+		pipelinesDir = flags.String(
+			"pipelines.path",
+			"./pipelines",
+			"path to the directory that has the yaml pipeline configuration files, or a single pipeline configuration file",
+		)
+		pipelinesStrictMode = flags.Bool(
+			"pipelines.strictMode",
+			false,
+			"strict mode for pipelines (Conduit shuts down if a pipeline fails)",
```

Comment:
is strict mode only for the pipelines provisioned by config? if so, we should mention it here

Reply:
Agreed!